packages/governance/src/contractHelper.js

```diff
@@ -159,10 +185,18 @@ const facetHelpers = (zcf, paramManager) => {
   };
```

Comment:
The `Far` on line 167 above encloses a record of kind methods --- methods that take a context as their first argument. This cannot be correct. If this is supposed to be a record of kind methods, ultimately to be used in a define*Kind call rather than a far class, then you should use `harden` rather than `Far`. You should also flag wherever this methods get passed around, because as we shift from kinds to far classes, we'll want to change all of these.

Reply:
I'm now using the version of those methods that does NOT use the context argument.